docs/topics/developer-tools.rst

```diff
@@ -254,15 +254,40 @@ page-number greater than 10), we increment the ``page`` attribute
 and ``yield`` a new request, inserting the incremented page-number 
 into our ``url``. 
 
-You can see that with a few inspections in the `Network`-tool we 
+In more complex websites, it could be difficult to easily reproduce the
+requests, as we could need to add ``headers`` or ``cookies`` to make it work. In those
+cases you can export the requests in `cURL <https://curl.haxx.se/>`_ format, by
+right-clicking on it in the network tool and using the
+:meth:`~scrapy.http.Request.from_curl()` method to generate an equivalent
+request. To do so, just import the :class:`~scrapy.http.Request` class and use this
```

Comment:
Since you are already linking to the reference documentation of `from_curl`, where users can check how to import `Request`, and moreover the example already covers how to import `Request` as well, I think the last sentence of this paragraph can be omitted (i.e. end with `to generate an equivalent request::`).